specification/compute/Gallery.Management/models.tsp

```diff
@@ -1786,6 +1850,195 @@ model GallerySoftDeletedResourceProperties {
+@added(Versions.v2025_03_03)
+model GalleryScriptUpdate extends UpdateResourceDefinition {
+  /**
+   * Gallery script definition properties to update.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-legacy-usage" "Backward compatibility"
+  @Azure.ClientGenerator.Core.Legacy.flattenProperty
+  properties?: GalleryScriptProperties;
+}
+
+/**
+ * Describes the properties of a gallery script version.
+ */
+@added(Versions.v2025_03_03)
+model GalleryScriptVersionProperties {
+  /**
+   * The publishing profile of a gallery image version.
+   */
+  publishingProfile: GalleryScriptVersionPublishingProfile;
+
+  /**
+   * The safety profile of the Gallery Script Version.
+   */
+  safetyProfile?: GalleryScriptVersionSafetyProfile;
+
+  /**
+   * The provisioning state, which only appears in the response.
+   */
+  @visibility(Lifecycle.Read)
+  provisioningState?: GalleryProvisioningState;
+
+  /**
+   * This is the replication status of the gallery image version.
+   */
+  @visibility(Lifecycle.Read)
+  replicationStatus?: ReplicationStatus;
+}
+
+/**
+ * The publishing profile of a gallery image version.
+ */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "For backward compatibility"
+@added(Versions.v2025_03_03)
+model GalleryScriptVersionPublishingProfile
+  extends GalleryArtifactPublishingProfileBase {
+  /**
+   * The source script from which the Script Version is going to be created.
+   */
+  source: ScriptSource;
+}
+
+/**
+ * The safety profile of the Gallery Script Version.
+ */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "For backward compatibility"
+#suppress "@azure-tools/typespec-azure-resource-manager/no-empty-model" "By design. Properties will be added in future."
+@added(Versions.v2025_03_03)
+model GalleryScriptVersionSafetyProfile
+  extends GalleryArtifactSafetyProfileBase {}
+
+/**
+ * The source script from which the Script Version is going to be created.
+ */
+@added(Versions.v2025_03_03)
+model ScriptSource {
+  /**
+   * Required. The link of the source script, it must be a readable storage blob with SAS URI or publicly accessible URI or managed identity enabled.
+   */
+  scriptLink: string;
+
+  /**
+   * Optional. Any input parameters that needs to passed to the script and are accessed within the script for its execution.
+   */
+  parameters?: GalleryScriptParameter[];
+}
+
+/**
+ * The definition of a parameter that can be passed to a script of a Gallery Script Version.
+ */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "For backward compatibility"
+@added(Versions.v2025_03_03)
+model GalleryScriptParameter extends GenericGalleryParameter {
+  /**
+   * Specifies the type of the Gallery Script parameter. Possible values are: String, Int, Double, Boolean, Enum
+   */
+  type?: GalleryScriptParameterType;
+
+  /**
+   * The minimum value of parameter.
+   */
+  minValue?: string;
```

Comment:
>minValue [](http://example.com/codeflow?start=2&length=8)

use full string, minimumValue

Reply:
This seems to be a new property being introduced in this PR right. Why will existing clients break this ?

Reply:
Yes, already deployed